doc/docs/master/reference/pipeline_spec.md

```diff
@@ -410,6 +416,17 @@ datums, and execution time per datum when setting this value. Keep in
 mind that the number of datums may change over jobs. Some new commits may
 have a bunch of new files (and so new datums). Some may have fewer.
 
+### S3 Output Repository
+
+`s3_out` defines whether to store the Pachyderm output repository
+in an S3 bucket in an S3 instance deployed in a sidecar container
+alongside the pipeline pod. When this parameter is set to `true`, Pachyderm
+includes a sidecar S3 gateway instance
+container in the same pod as the pipeline container. The address of the
```

Comment:
```suggestion
`s3_out` allows your pipeline code to write results out to an S3 gateway endpoint instead of the typical `pfs/out` directory. When this parameter is set to `true`, Pachyderm
includes a sidecar S3 gateway instance
container in the same pod as the pipeline container. The address of the
```

Reply:
added